game.py

```diff
@@ -86,12 +92,23 @@ def is_legal_move(self,current_player, move):
 		return True
 
 	# seqence when it is player's turn
-	def player_turn():
-		pass
-
-	# sequence when it is computer's turn
-	def computer_turn():
-		pass
+	def player_turn(self):
+		is_valid_input = False
+		while is_valid_input == False:
+			move_coordinates = (input("Please enter start coordinate: "), input("Please enter end coordinate: "))	# should be two tuples entered
```

Comment:
We don't actually need to break it up into start and end, we could just input it as `((1,1),(1,3))`. Splitting might be clearer though? I'm not sure